src/parser.js

```diff
@@ -5,7 +5,7 @@ const { forEach } = require("lodash");
 
 const FULL_SOURCE = "source/source-full.md";
 const CONTENT_SOURCE = "source/source.md";
-const {HASH_TAG, LINK_HTTPS, LINK, STR_BEGIN, STR_END} = require("./constants.js");
+const {REGEXP_HASH_TAG, REGEXP_LINK_HTTPS, REGEXP_LINK_G, REGEXP_STR_BEGIN, REGEXP_STR_END} = require("./constants.js");
```

Comment:
long line

Reply:
Should we divide it or just change constants?